src/main/java/com/flightstats/hub/spoke/SpokeWriteContentDao.java

```diff
@@ -58,11 +58,12 @@ public ContentKey insert(String channelName, Content content) {
     }
 
     @Override
-    public SortedSet<ContentKey> insert(BulkContent bulkContent) throws Exception {
-        return SpokeContentDao.insert(bulkContent, (baos) -> {
+    public SortedSet<ContentKey> insert(BulkContent bulkContent) {
+        SortedSet<ContentKey> keys = SpokeContentDao.insert(bulkContent, (baos) -> {
             String channel = bulkContent.getChannel();
             return clusterWriteSpoke.insertToWriteCluster(channel, baos.toByteArray(), "bulkKey", channel);
         });
+        return keys;
```

Comment:
Minor nit as there's no harm here, but IntelliJ should warn you're setting and returning the variable immediately and offer to reduce this to a single line. If you're doing it in 2 lines so you can set a breakpoint on the return, IntelliJ's debugger has a feature where it'll show you the return value of any method you just exited, which basically accomplishes the same thing.

Reply:
Yep, for sure.  There was more code here at one point.